MetaMask/metamask-extension
# Update Q Mainnet symbol

## **Description**

According to https://github.com/ethereum-lists/chains/pull/4389 Q Mainnet symbol is changed from "Q" to "QGOV". This change is not reflected in MetaMask and the users see the scam badge on their Q Mainnet balance.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/PR?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to https://chainid.network/
2. Search for 35441 (Q Mainnet chain ID)
3. Add network to MetaMask
4. Switch to Q Mainnet network in MetaMask
5. Notice the scam warning in asset list

## **Screenshots/Recordings**



<img width="349" alt="Screenshot 2024-02-21 at 18 58 42" src="https://github.com/MetaMask/metamask-extension/assets/17788043/5895818b-c164-447e-8f2c-3ed5417a5dc8">
<img width="352" alt="Screenshot 2024-02-21 at 18 58 48" src="https://github.com/MetaMask/metamask-extension/assets/17788043/85840e3f-b1e3-47f5-874e-5036534d1527">

### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
